assets/css/sass/partials/pages/_modeling.scss

```diff
@@ -863,6 +863,12 @@ $blue-to-red: #2791c3 10%,
         }
     }
 
+    .new-plan {
+        float: left;
+        margin-top: 21px;
+        margin-left: 30px;
```

Comment:
I'd like to consider calling more attention to the "New" button by styling it with the primary button color.

```sass
background-color: $button-primary;
border-color: $button-primary;
color: $paper;
```

It calls more attention to this important action

<img width="818" alt="screen shot 2016-10-03 at 9 27 00 am" src="https://cloud.githubusercontent.com/assets/17363/19045226/7fb4e8d4-894c-11e6-97ec-a68820b19be2.png">


<img width="822" alt="screen shot 2016-10-03 at 9 21 25 am" src="https://cloud.githubusercontent.com/assets/17363/19045243/9a4af3dc-894c-11e6-8e6f-30650fdcf1ef.png">